### ISSUE TITLE
Downloads Folder needs a broader access permission available

### ISSUE THREAD

**Issue by michael-hawker** [microsoft/WindowsAppSDK#1383]
There's also some weird restrictions on existing capabilities, like access to the `Downloads` folder only lets you create a sub-directory, see [doc issue here](https://github.com/MicrosoftDocs/winrt-api/issues/386). If you're trying to make an alternate browser, this is a weird experience for anyone using any other browser. Maybe there needs to be a `AllDownloads` capability that's slightly broader?

_Originally posted by @michael-hawker in https://github.com/microsoft/WindowsAppSDK/issues/8#issuecomment-661432481_

**Comment by ptorr-msft** [comment]
Tracking as a feature request. The problem is well-understood.

**Comment by mominshaikhdevs** [comment]
* #8.

**Comment by michael-hawker** [comment]
@mominshaikhdevs yes, I referenced that issue in the original post. It was asked to separate this request out separately, as it was a more specific actionable request:

> This is a missing feature (downloads folder capability) -- please add another issue for it. Thanks!
